packages/server/providers/couchdb.js

```diff
@@ -224,7 +224,7 @@ const couchDb = (app, io, pouchOptions) => {
       res.status(404).send({ msg: 'Wrong Player Name', data: null })
     }
 
-    const db = new CouchDB(couchDbURL(databaseName))
+    const db = new PouchDB(databaseName, pouchOptions)
```

Comment:
@lilitkarapetyan - shouldn't this use `CouchDB` ?

Reply:
Hello Ian, yes we should use CouchDB.